go/worker/common/p2p/types.go

```diff
@@ -22,11 +22,6 @@ type Message struct {
 	// Jaeger's span context in binary format.
 	SpanContext []byte `json:"span,omitempty"`
 
-	// Message types.
-
-	Ack   *Ack   `json:"ack,omitempty"`
-	Error *Error `json:"err,omitempty"`
-
 	SignedTxnSchedulerBatchDispatch *commitment.SignedTxnSchedulerBatchDispatch `json:",omitempty"`
```

Comment:
While we're changing this could we do some cleanup in this structure:
* Maybe remove `RuntimeID` as it is now part of the topic.
* Rename `SignedTxnSchedulerBatchDispatch` to `TxnSchedulerBatch` (the field and the commitment type).
* Rename `ExecutorWorkerFinished` to `ExecutorCommit` and remove the intermediate wrapper struct (`ExecutorWorkerFinished`), using `commitment.ExecutorCommitment` directly instead.